web/pages/[username]/calibration.tsx

```diff
@@ -205,8 +205,27 @@ const getCalibrationPoints = (betsData: [Contract, LimitBet[]][]) => {
     if (resolution !== 'YES' && resolution !== 'NO') continue
     const resolvedYES = resolution === 'YES'
 
-    for (const bet of bets as Bet[]) {
-      if (bet.amount < 0) continue // skip sales
+    let currentPosition = 0
+
+    // bets is reversed in place to track user current position
+    for (const bet of bets.reverse() as Bet[]) {
```

Comment:
.reverse() is a mutable method, which could lead to subtle errors in the future. Prefer .slice().reverse()